src/app/handler/user/user.handler.go

```diff
@@ -34,27 +50,102 @@ func (h *Handler) FindOne(c *router.FiberCtx) {
 		return
 	}
 
-	c.JSON(http.StatusCreated, user)
+	c.JSON(http.StatusOK, &dto.ResponseSuccess{
+		StatusCode: http.StatusOK,
+		Message:    userconst.FindOneUserSuccessMessage,
+		Data:       user,
+	})
 	return
 }
 
-func (h *Handler) Update(c *router.FiberCtx) {
+// Update is a function that updates user in database
+// @Summary updates user
+// @Description Returns the data of user if successfully
+// @Param update body dto.UpdateUserRequest true "update user dto"
+// @Tags auth
+// @Accept json
+// @Produce json
+// @Success 201 {object} dto.User
+// @Failure 400 {object} dto.ResponseBadRequestErr "Invalid request body"
+// @Failure 500 {object} dto.ResponseInternalErr "Internal service error"
+// @Failure 503 {object} dto.ResponseServiceDownErr "Service is down"
+// @Router /v1/users [put]
+func (h *Handler) Update(c router.IContext) {
 	usrId := c.UserID()
 
```

Comment:
sorry for lately catch up, why we extract user id from the context, not the path.

Reply:
because user should not be able to update other users

Reply:
so the logged in user's id stored in context is used

Reply:
ok